build-support/bin/bootstrap_pants_pex.sh

```diff
@@ -58,6 +58,7 @@ function calculate_pants_pex_current_hash() {
      "${REPO_ROOT}/build-support/regexes" \
      "${REPO_ROOT}/build-support/scalafmt" \
      "${REPO_ROOT}/build-support/scalastyle" \
+     "${REPO_ROOT}/build-support/source_roots" \
```

Comment:
Now stale because of a major simplification in https://github.com/pantsbuild/pants/pull/9798.